Tailwind Portfolio/index.html

```diff
@@ -1,6 +1,6 @@
 
 <!DOCTYPE html>
-<html lang="en">

```

Comment:
this files should not be present in this PR

Reply:
Yeh cheeze toh pehel hi thi usme

Reply:
Done , Please aab check karo @arpit456jain @VidhiBhatt01